certbot/certbot/configuration.py

```diff
@@ -14,6 +17,45 @@
 from certbot.compat import os
 
 
+# Maps a config option to a set of config options that may have modified it.
+# This dictionary is used recursively, so if A modifies B and B modifies C,
+# it is determined that C was modified by the user if A was modified.
+VAR_MODIFIERS = {"account": {"server",},
```

Comment:
Why is this copied here instead of pulling from `cli_constants`? Same with `DEPRECATED_OPTIONS`

Reply:
ah shoot i meant to de-uglify this. i had to remove them from their `_internal` home to avoid a circular dependency, but i'd be happy to extract them to `certbot/certbot/constants.py` or something similar

Reply:
why is it a circular dependency? looks like `cli_constants.py` doesn't import anything

Reply:
right you are lol, idk what was circularizing before but it's not anymore. reverting!

Reply:
`cli_constants.py` doesn't, but `cli/__init__.py` imports `NamespaceConfig` now

Reply:
hm, does it work if we defer the import until the class is created? if not yeah let's just stick them somewhere else. basically good with whatever as long as they're not defined twice.